ironfish/src/assets/assetsVerificationApi.ts

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import axios, { AxiosError } from 'axios'
+
+type GetVerifiedAssetsResponse = {
+  data: Array<{ identifier: string }>
+}
+
+type GetVerifiedAssetsRequestHeaders = {
+  'if-modified-since'?: string
+}
+
+type GetVerifiedAssetsResponseHeaders = {
+  'last-modified'?: string
+}
+
+export class VerifiedAssets {
+  private readonly assetIds: Set<string> = new Set()
```

Comment:
It feels like this should be in the verifier, rather than the API since the verifier just consumes the API to hydrate its internal stores.

Also I commented on the spec, but it feels like later we should persist this with a FileStore in the data directory along with the last fetch time. If we did want to do this, we can do it as an expansion.

Reply:
Yes, this class is meant to be persisted (a future PR will take care of that). Not sure if I it makes sense to move this class to the verifier, because (1) it's the return value of the API class, and (2) it's crucial to the logic of `refreshVerifiedAssets` from the API class, so I'd say it belongs to the API module.